mica/report/report.py

```diff
@@ -117,7 +118,7 @@ def rec_to_dict_list(recarray):
 
 
 def get_star_acq_stats(id):
-    hdu = tables.open_file(acq_stats.table_file)
+    hdu = tables.open_file(acq_stats.TABLE_FILE)
```

Comment:
What's happening with this change?

Reply:
I don't remember when the name changed in acq stats.

Reply:
In retrospect, it is probably https://github.com/sot/mica/pull/173/files but still just need to get the reference to match whatever it is by the time we promote this code.